ui/components/onboarding/CitizenImageGenerator.tsx

```diff
@@ -98,11 +99,14 @@ export function ImageGenerator({
       {inputImage && (
         <StageButton
           onClick={() => {
-            setImage(null)
-            generateImage()
-            if (generateInBG) {
-              nextStage()
-            }
+            toast.error(
```

Comment:
Hopefully we can figure out an alternative soon. Thanks for getting this patched